### ISSUE TITLE
Create Scanner Assigned Pages and Versions System Diagram

### ISSUE THREAD

**Issue by weatherpattern** [edgi-govdata-archiving/web-monitoring#107]
Create visualizations of how the various modules for displaying Scanner's Assigned Pages and Versions System Diagrams work together. They will be published in the github project documentation.

Each page will have its own diagram.
Drafts: https://scene.zeplin.io/project/5ad6aa451f6e994b2acd83da
(For access to zeplin to allow for annotation contact @weatherpattern )

The Assigned Pages:
Step 1. Analysts requests Assigned Pages (from a browser)
Step 2. Frontend calls UI Backend.
Step 3. Get Analyst's Assigned Domains from Google Docs
Step 4. UI Backend gets Pages metadata from API and sends it to the UI Frontend.
Step 5. Display Assigned Pages.

Versions view (first time):
Step 1. Analyst Requests the Diff of two Versions for the first time
Step 2. UI call to API to fetch Versions metadata
Step 3. API gets Versions metadata
Step 4. DB sends Versions metadata
Step 5. UI immediately requests the default diff
Step 6. API sends metadata including URLs (in S3) to diff
Step 7. Diffing Service gets URL content of Versions from AWS S3 
Step 8. Diffing Service sends diff content back to API
Step 9. API sends diff content back to UI
Step 10. UI displays default diff of two Versions

New Diff Versions View (may not be needed)
Step 1. Analyst Requests new Diff View of Versions currently displayed
Step 2. UI requests the new diff
Step 3. API sends metadata including URLs (in S3) to diff
Step 4. Diffing Service gets URL content of Versions from AWS S3
Step 5. Diffing Service sends diff content back to API
Step 6. API sends diff content back to UI     
Step 7. UI displays new diff of two Versions

**Comment by weatherpattern** [comment]
Thanks!

Drafts: https://scene.zeplin.io/project/5ad6aa451f6e994b2acd83da
I added a new version: v04.

For Assigned Pages:
Step 4
> The UI frontend actually contacts the API directly. 
@danielballan  and I weren't sure of this. Thanks for clarifying. The UI frontend now connects directly to the API. 
Step 5:
I added first available, just in case it's helpful for future readers.
Step 6:
Changed it to " API requests diff from diffing server"

Step 7:
Clarified that S3 is Versionista specific.

Finally:
- I removed New Diff Versions. @lightandluck had suggested the same thing.
